pt-BR/mail/chrome/messenger/pbMessenger.dtd

```diff
@@ -138,223 +138,223 @@
 <!ENTITY pbAccountTreeContextMenu-newGroup.accesskey "G">
-<!ENTITY pbAccountTreeContextMenu-settings.label "Settings...">
+<!ENTITY pbAccountTreeContextMenu-settings.label "Configurações...">
 <!ENTITY pbAccountTreeContextMenu-settings.accesskey "i">
 
 <!-- folder pane -->
-<!ENTITY accountTreeMac.label    "ACCOUNTS">
-<!ENTITY folderTreeMac.label     "FOLDERS">
+<!ENTITY accountTreeMac.label    "CONTAS">
+<!ENTITY folderTreeMac.label     "PASTAS">
 
 <!-- thread pane -->
-<!ENTITY threadPaneNoMatchesFound.desc "No messages matched your search.">
-<!ENTITY threadPaneNoMatchesSuggestionAllFolders.desc "Try looking in All Folders.">
-<!ENTITY threadPaneNoMatchesSuggestionAllFocus.desc "Try setting your Focus to All.">
-<!ENTITY threadPaneNoMatchesSuggestionAllMessages.desc "Try setting your view to Messages.">
-<!ENTITY threadPaneAddTextToSearch.desc "Start typing in the search bar to begin your search.">
+<!ENTITY threadPaneNoMatchesFound.desc "Nenhuma mensagem corresponde aos critérios da pesquisa.">
+<!ENTITY threadPaneNoMatchesSuggestionAllFolders.desc "Tente verificar em Todas as Pastas.">
+<!ENTITY threadPaneNoMatchesSuggestionAllFocus.desc "Tente definir seu foco para Todos.">
+<!ENTITY threadPaneNoMatchesSuggestionAllMessages.desc "Tente definir sua exibição para Mensagens.">
+<!ENTITY threadPaneAddTextToSearch.desc "Comece a digitar na barra de pesquisa para iniciar sua pesquisa.">
 
-<!ENTITY pbSignatureEditor.title "New Signature">
-<!ENTITY pbSignatureEditor.name "Name">
-<!ENTITY pbSignatureEditor.body "Signature">
+<!ENTITY pbSignatureEditor.title "Nova assinatura">
+<!ENTITY pbSignatureEditor.name "Nome">
+<!ENTITY pbSignatureEditor.body "Assinatura">
 
 <!-- pbSearchBox.xml -->
-<!ENTITY pbSearchScopePanel-location.label "Location">
-<!ENTITY pbSearchScopePanel-recent.label "Search History">
+<!ENTITY pbSearchScopePanel-location.label "Caminho">
+<!ENTITY pbSearchScopePanel-recent.label "Pesquisar histórico">
 
 <!-- facebook extension login.xul and auth.xul -->
-<!ENTITY login.window.title "Login to Facebook">
-<!ENTITY login.wait "Please wait while we prepare to log you in...">
-<!ENTITY login.close "Close login window">
-<!ENTITY auth.window.title "Allow Status Updates from Postbox">
-<!ENTITY auth.close "Close window">
+<!ENTITY login.window.title "Fazer login no Facebook">
+<!ENTITY login.wait "Por favor, aguarde enquanto preparamos o processo de login...">
+<!ENTITY login.close "Fechar janela de login">
+<!ENTITY auth.window.title "Permitir atualizações de status pelo Postbox">
+<!ENTITY auth.close "Fechar janela">
 
 <!-- pbLanguageProgress.xul -->
-<!ENTITY downloadLanguage.title "Download Language">
+<!ENTITY downloadLanguage.title "Baixar idioma">
 
 <!-- thread pane context menu -->
-<!ENTITY pbThreadPaneContext-omniFocusMac.label      "OmniFocus: Send to Inbox">
+<!ENTITY pbThreadPaneContext-omniFocusMac.label      "OmniFocus: enviar à Caixa de Entrada">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY pbThreadPaneContext-omniFocusMac.label      "OmniFocus: enviar para a Caixa de Entrada">